File: service-batch/src/main/java/com/issuemoa/batch/domain/store/Store.java
package com.issuemoa.batch.domain.store;

import lombok.*;
import org.springframework.data.annotation.CreatedDate;
import javax.persistence.*;
import java.time.LocalDateTime;

@ToString
@AllArgsConstructor
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@Getter
@Builder
@Table(name = "stores")
@Entity
public class Store {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String entpId;
    private String name;
    private String typeCode;
    private String areaCode;
    private String detailCode;
    private String tel;
    private String postNo;
    private String addr;
    private String addrDetail;
    private String roadAddr;
    private String roadAddrDetail;
    private String xCoord;
    private String yCoord;
    private Long registerId;
    private Long modifyId;
    private LocalDateTime registerTime;
    private LocalDateTime modifyTime;
}

File: service-store/src/main/kotlin/com/issuemoa/store/entity/Store.kt
package com.issuemoa.store.entity

import javax.persistence.*
import java.time.LocalDateTime

@Entity
@Table(name = "stores")
data class Store(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long = 0,
    val entpId: Long = 0,
    val name: String,
    val typeCode: String,
    val areaCode: String,
    val detailCode: String,
    val tel: String,
    val postNo: Int? = null,
    val addr: String,
    val addrDetail: String,
    val roadAddr: String,
    val roadAddrDetail: String,
    val xCoord: Int? = null,
    val yCoord: Int? = null,
    val registerId: Long,
    val modifyId: Long? = null,
    val registerTime: LocalDateTime,
    val modifyTime: LocalDateTime? = null
)